x-pack/plugins/cases/public/components/user_action_tree/user_action_host_isolation_comment_event.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useCallback } from 'react';
+import * as i18n from './translations';
+import { LinkAnchor } from '../links';
+import { useKibana } from '../../common/lib/kibana';
+import { SECURITY_SOLUTION_APP_ID } from './constants';
+
+interface Props {
+  type: string;
+  hostId: string;
```

Comment:
This will be an array,  Show the first Endpoint as a link.  In the case where there are multiple, show "and [x] more".